File: lending-backend/internal/model/media.go
package model

type Media struct {
	MediaID     int    `json:"media_id"`
	OrderID     *int   `json:"order_id,omitempty"`
	ObjectID    int    `json:"object_id"`
	Type        string `json:"type"`
	Description string `json:"description"`
	URL         string `json:"url"`
	Link        string `jons:"link"`
	CreatedAt   string `json:"created_at"`
}

type CreateMediaRequest struct {
	OrderID     *int   `json:"order_id,omitempty"`
	ObjectID    int    `json:"object_id"`
	URL         string `json:"url"`
	Type        string `json:"type"`
	Link string `json:"link"`
	Description string `json:"description"`
}

File: lending-backend-dotnet/src/LendingSystem.Domain/Media/Media.cs
namespace LendingSystem.Domain.Media;

public static class MediaTypes
{
    public const string Image = "image";
    public const string Video = "video";
}

public sealed record MediaAsset(
    int MediaId,
    int? OrderId,
    int ObjectId,
    string Type,
    string Description,
    string Url,
    string Link,
    DateTimeOffset CreatedAt);
